### PR TITLE
IT-5547 | LoadingBar

### PR DESCRIPTION
**Pull request by pioziol123**

### CODE REVIEW

File: lib/js/components/Stripe/Stripe.vue [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	}
+
+	&.-ds-size-medium {
+		height: 6px;
+	}
+
+	&__loader {
+		$STRIPE_COLORS: (
+			neutralheavy: $color-neutral-border-heavy,
+			neautralstrong: $color-neutral-border-strong,
```
Comment (rogatty) [inline lib/js/components/Stripe/Stripe.vue:32]: Typo.

File: lib/js/components/StripeLoadder/StripeLoader.vue [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+		color: {
+			type: String as PropType<StripeLoaderColors>,
+			default: STRIPE_LOADER_COLORS.NEUTRAL_HEAVY,
+		},
+		size: {
+			type: String as PropType<StripeLoaderSizes>,
+			default: STRIPE_LOADER_SIZES.MEDIUM,
+		},
+		time: {
+			type: String,
```
[the remark this answers is not in the document]
Comment (pioziol123) [reply lib/js/components/StripeLoadder/StripeLoader.vue:80]: We defined the prop this way in miro

[the remark this answers is not in the document]
Comment (tweetgeek) [reply lib/js/components/StripeLoadder/StripeLoader.vue:80]: I don't remember why it was saved this way, but it makes a bit of sense to save time as a string

File: lib/js/components/Cards/CardExpandable/CardExpandable.vue [hunk 3]
```diff
@@ -104,9 +104,7 @@ export default {
 			default: false,
 		},
 	},
-	// TODO fix me when touching this file
-	// eslint-disable-next-line vue/require-emit-validator
-	emits: ['update:isExpanded'],
+	emits: { 'update:isExpanded': (isExpanded: Boolean): Boolean => isExpanded },
```
[the remark this answers is not in the document]
Comment (pioziol123) [reply lib/js/components/Cards/CardExpandable/CardExpandable.vue]: fixed

**Review by tweetgeek — APPROVED** [review]

**Pull request merged**